cmake/QskConfigMacros.cmake

```diff
@@ -163,3 +163,23 @@ macro(qsk_finalize_build_flags)
     endif()
 
 endmacro()
+
+function(qsk_internal_sync_generated_target_folder directory)
+    # loop through all targets in the directory recursively and
+    # synchronize the generated target's folder property with 
+    # the actual target's folder property
+    get_property(TARGETS DIRECTORY "${directory}" PROPERTY BUILDSYSTEM_TARGETS)
+    foreach(target IN LISTS TARGETS)
+        if(TARGET ${target}_qmlimportscan)
+            get_target_property(folder ${target} FOLDER)
+            set_target_properties(${target}_qmlimportscan PROPERTIES FOLDER ${folder})
```

Comment:
@uwerat we could also define a collective "generated" "folder" for the IDE's to hide these implementation / Qt / CMake specific targets e.g.:

![image](https://user-images.githubusercontent.com/6728289/230920613-74a7e126-5054-4fbb-8601-371e1aeb7b6b.png)

Reply:
PS: should we put the `qskqmlexport` + `qsktestsupport` targets into the `libs` folder aswell?